### ISSUE TITLE
IONIQ EV 2020 - LFA Warning on the dash when OP is running

### ISSUE THREAD

**Issue by dermida** [commaai/openpilot#29552]
### Describe the bug

When OpenPilot is running the car shows on the dash a warning to "Check LFA system".

![IMG20230822213320](https://github.com/commaai/openpilot/assets/85565557/6aae5c73-9c01-413f-8743-83432078d3e5)

My car only has "LKA", but now thinks it has LFA and the warning appeared.

![rn_image_picker_lib_temp_c3fea711-ec6c-47eb-92df-45fbc7fbe969](https://github.com/commaai/openpilot/assets/85565557/7e76dd17-2183-4d04-b6c3-3d449fc6733f)

This warining started to appear now with the C3X, before with C2 didn't appear.

With the help of @sshane we were able to identify that: "When we used to send an LFA message to your car, which normally doesn't have LFA, it put the car into a persistent state where it thinks it has LFA, so our missing message is causing a fault"

Discord Thread: [car thinks it has LFA](https://discord.com/channels/469524606043160576/1143645816536436746)

### Which car does this affect?

HYUNDAI IONIQ ELECTRIC 2020

### Provide a route where the issue occurs

085e33b2085c44f4|2023-08-22--08-08-05

### openpilot version

0.9.5-lfa-msg-test (606cb6547c567582924c08df23bdc26e18a70524)

### Additional info

With the help of @sshane we tested a branch (lfa-msg-test (606cb6547c567582924c08df23bdc26e18a70524)) that forces sending de "LFA" message and with that branch all warnings disappeared

![IMG20230822224229](https://github.com/commaai/openpilot/assets/85565557/7b85a497-a3ae-4adf-b086-2883af27ae4d)

**Comment by adeebshihadeh** [comment]
@sshane is this still an issue?

**Comment by sshane** [comment]
Yes, the car (cluster, ABS?) is expecting the LFA message but is not receiving it because we detect the camera not sending it.

@dermida do the warning show up without the comma 3X plugged in? What about if the 3X is in dashcam mode? Can you get a route where it's in dashcam mode?

**Comment by dermida** [comment]
> @dermida do the warning show up without the comma 3X plugged in? 
Yes. Even when not plugged in.

> What about if the 3X is in dashcam mode? 
Never tried. But i can try if you need.

> Can you get a route where it's in dashcam mode?
Sure. I will try tomorrow.

**Comment by sshane** [comment]
Sounds like we put one of your ECUs into a state of expecting the LFA message before we detected it in https://github.com/commaai/openpilot/pull/27323. So, the current openpilot behavior is not a bug, and I don't think there's a solution to this that should be merged. I would recommend trying to visit your dealer to have them check it out and re-flash the ECU responsible.

The best we can do is try to narrow down the message that describes the fault, then we can have a good guess of which ECU we affected in the past. Feel free to reach out on Discord.
